RADAR_LEADIN_ROUTINES.s

```diff
@@ -49,7 +49,7 @@ RADSAMP		CCS	RSAMPDT		# TIMES NORMAL ONCE PER SECOND SAMLING
 		EBANK=	RSTACK
 		2CADR	DORSAMP
 		
-		CAF	BIT14		# FOR CYCLIC SAMPLING, RTSTDEX=
+		CAF	BIT14		# FOR CYCLIC SAMPLING, RTSTDEX
```

Comment:
Actually, there should be a space: "RTSTDEX ="
http://www.ibiblio.org/apollo/ScansForConversion/Luminary099/0490.jpg